terraform/aws/vpc/main.tf

```diff
@@ -0,0 +1,22 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "2.15.0"
+
+  name               = "${var.cluster_name}-vpc"
+  cidr               = "10.18.0.0/16"
```

Comment:
Hardcode.

Good idea move it to variable.

Reply:
done